src/java/voldemort/client/protocol/admin/AdminClient.java

```diff
@@ -3305,6 +3300,22 @@ public void putNodeKeyValue(String storeName, NodeValue<ByteArray, byte[]> nodeK
         }
 
         /**
+         * Fetch values for given keys on a specific store present on a specific node.
+         * 
+         * @param storeName Name of the Store
+         * @param nodeId Id of the node to query from
+         * @param keys List of keys to be required.
+         * @return
+         * Only keys that has values will be returned in a Map.
+         * Keys that does not exists will not be present in the Map.
+         */
+        public Map<ByteArray, List<Versioned<byte[]>>> getAllNodeKeys(String storeName, int nodeId,
```

Comment:
Ok, I wasn't aware of the convention with the other APIs. We can leave it as is then.